README.md

```diff
@@ -111,7 +111,9 @@ To run the bot in simulation mode, try this command:
 
 _Note for Windows Users: You should use a [Bash Shell][bash] to follow the steps below. This will give you a UNIX environment in which to run your commands and will enable the `./scripts/build.sh` bash script to work correctly._
 
-To compile Kelp from source:
+_Note for MacOS Users: Running [install-macos.sh][install-macos-script] should automate the steps below. (including cloning the Kelp repo to the correct Go folder) Remember, manual installation of PostgreSQL and Docker are additionally required._
+
+## Manual Installation Steps
```

Comment:
can you add a section called "Automated Install for MacOS" above this section on "Manual Installation Steps"?
Then you can include the text you have in the note above as part of that section. That will give more importance to your automated script.